src/components/TagsSection/Tag.tsx

```diff
@@ -116,6 +116,7 @@ export const Tag = ({
           onChange({ ...tag, operator: op.value })
         }}
       />
+      {operator === 'IS NULL' || operator === 'IS NOT NULL' || operator === 'EXISTS' || operator === 'NOT EXISTS' ? null : (
```

Comment:
Could we define a list of operators that don't have a value to have it a bit less hardocded here?
I know it will only move the check but somehow it feels cleaner.